config/configtls/configtls.go

```diff
@@ -26,6 +26,9 @@ const defaultMaxTLSVersion = 0
 // Note: Since there isn't anything specific to a server connection. Components
 // with server connections should use TLSSetting.
 type TLSSetting struct {
+	// If true, load system certificates in addition to the certificates
+	// configured in this struct.
+	UseSystemCACerts bool `mapstructure:"use_system_ca_certs"`
```

Comment:
The name implies to me that we are going to use the System CA Certs instead of other options supplied.  How about `IncludeSystemCACerts` or maybe something with `Pool` in the name.

Reply:
done!